emails/Events.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import Events from '../src/app/(payload)/components/email/Events'
+
+export function EventsEmail() {
```

Comment:
this directory is what runs when you run `npm run dev:email`.

I made it consume the Events email component in payload that way we can get a preview with relevant-ish content when running locally and making changes

Reply:
Should we remove this from the commit before merging?